modules/phase_field/src/materials/ComputePolycrystalElasticityTensor.C

```diff
@@ -48,9 +48,6 @@ ComputePolycrystalElasticityTensor::computeQpElasticityTensor()
   const std::vector<std::pair<unsigned int, unsigned int> > & active_ops = _grain_tracker.getElementalValues(_current_elem->id());
```

Comment:
Yeah, I understand. `getOpToGrainsVector` is cleaner and requires less work to obtain grain numbers, but I am not sure how efficient it would be when we have to loop over all the order parameters (considering this also will need additional check for valid grain no.). What's you opinion @dschwen?